pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[build-system]
+requires = [
+    "setuptools",
+    "wheel",
+]
+
+[tool.black]
+line-length = 79
+target-version = [
+    'py36',
+    'py37',
+]
+include = '\.pyi?$'
+exclude = '''
+/(
+    .*\.egg-info
+  | .*build
```

Comment:
Is build also necessary? I'm not sure setuptools will create something like this?

Reply:
Hmm ok sphinx will create a build dir. But I have to read up on how to read this syntax here

Reply:
I'm not sure, it came from black's [pyproject.toml example](https://black.readthedocs.io/en/stable/pyproject_toml.html) originally, then left it in for skbuild in pycompwa. I don't think it hurts to leave it in.

Reply:
> I have to read up on how to read this syntax here

It's regex (only not sure about the `|`).

Reply:
black default for exclude is:
[default: /(\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|_build|buck-out|build|dist)/]

Not sure why they did not make a `.*` for build

Reply:
> It's regex (only not sure about the |).

I assume its logical or

Reply:
I guess its fine. I can't think of any reason why `.*build` should cause problems. We can also change it later on if it does

Reply:
I added the regex `.*` for 'anything preceding it'